app/questionnaire/questionnaire_manager.py

```diff
@@ -1,19 +1,27 @@
-import json
 import logging
 
-from app.globals import get_answers, get_questionnaire_store
-from app.piping.plumbing_preprocessor import PlumbingPreprocessor, get_schema_template_context
+from app.globals import get_answers, get_metadata, get_questionnaire_store
 from app.questionnaire.navigator import Navigator, evaluate_rule
-from app.templating.model_builder import build_questionnaire_model, build_summary_model
+from app.templating.schema_context import build_schema_context
+from app.templating.template_renderer import TemplateRenderer
 
-from flask import render_template_string
+from flask import g
 
 from flask_login import current_user
 
 logger = logging.getLogger(__name__)
 
 
+def get_questionnaire_manager(schema, schema_json):
```

Comment:
@iwootten I would have put this in globals but it introduces a circular dependency. Also at the moment the globals contains questionnaire store related stuff which is nice without this polluting it however not sure where else to put this. The other option is it can live in its own file